rollup.config.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import { getPackages } from '@lerna/project'
```

Comment:
the full rollup config. we are essentially rolling up all vendor deps and _this_ (reach package in turn) packing into a single file, and then externally importing all other @textile packages. if we find that some of our vendor deps also start supporting es modules, then we can add them to the external array here.

the benefit is that browsers and other env can in theory tree-shake and lazily load these a bit better. ESM is the future, so we're setup for that now